doc/en/user/source/services/wfs/reference.rst

```diff
@@ -316,7 +318,7 @@ An example request involving returning features based on bounding box would be i
      request=GetFeature&
      typeNames=namespace:featuretype&
      srsName=CRS
-     bbox=a1,b1,a2,b2
+     bbox=a1,b1,a2,b2,CRS
```

Comment:
Do we want two examples, so it really does not look like CRS is always required?

```
A second example request returning features based on the bounding requested, when it is different from the featureTypes native CFS)

   http://example.com/geoserver/wfs?
     service=wfs&
     version=2.0.0&
	     request=GetFeature&
	     typeNames=namespace:featuretype&
	     srsName=CRS
	     bbox=a1,b1,a2,b2,CRS
 
```